docs/production/authentication-methods.md

```diff
@@ -598,6 +598,34 @@ to the root and `engineering` subdomains:
          importing, only the certificate will be displayed (not the private
          key).
 
+
+### Using Authentik as a SAML IdP
+
+1. Make sure you reviewed [this article](https://goauthentik.io/integrations/services/zulip/), which
+   details how to integrate Zulip with Authentik.
+2. Verify that `SOCIAL_AUTH_SAML_ENABLED_IDPS[{idp_name}]['entity_id']` and
+   `SOCIAL_AUTH_SAML_ENABLED_IDPS[{idp_name}]['url']` are correct in your Zulip
+   configuration. Specifically, if `entity_id` is
+   `https://authentik.example.com/`, then `url`
+   should be
+   `https://authentik.company/application/saml/<application slug>/sso/binding/redirect/` where \<application slug\>
+  is the application slug you've assigned to this application in Authentik settings. (i.e 'zulip')
+3. Your Authentik public certificate must be saved on the Zulip server
+   as `{idp_name}.crt` in `/etc/zulip/saml/idps/`. You can obtain the
+   certificate from the Authentik ui in the `Certificates` section or directly 
+   from the provider page. Download the certificate and copy it to the server.
+   
+   (Alternatively, open the provider you created and copy the certificate embedded in the 
```

Comment:
Is "open the provider" some Authentic-specific phrasing? Should we perhaps just phrase this as "go to the metadata URL of the provider and ..." or something like that?

Reply:
Authentik's GUI has Providers. People who use it would get this but we can rephrase it. But it's not goig to the metadata url. It's actually a section in its management gui where you can open the provider settings (for a particular application) and click a button to download the certificate you associated with that provider. You can also go to the global Certificates section of the gui and download it there too.

Reply:
perhaps "open the provider settings page" would be clearer.

Reply:
Sounds good, I used this type of phrasing.